src/test/java/seedu/address/testutil/TypicalLocations.java

```diff
@@ -65,8 +65,21 @@ public static LocationBook getTypicalLocationBook() {
         return lb;
     }
 
+    public static LocationBook getTypicalLocationBookForTest() {
+        LocationBook lb = new LocationBook();
+        for (Location location : getLocationsForTest()) {
+            lb.addLocation(location);
+        }
+        return lb;
+    }
+
     public static List<Location> getTypicalLocations() {
         return new ArrayList<>(Arrays.asList(ALICE_LOCATION, BENSON_LOCATION, CARL_LOCATION, DANIEL_LOCATION,
                 ELLE_LOCATION, FIONA_LOCATION, GEORGE_LOCATION));
     }
+
+    public static List<Location> getLocationsForTest() {
+        return new ArrayList<>(Arrays.asList(IDA_LOCATION, BENSON_LOCATION, CARL_LOCATION, DANIEL_LOCATION,
+                ELLE_LOCATION, FIONA_LOCATION, GEORGE_LOCATION));
+    }
```

Comment:
This method seems very similar to the above method getTypicalLocations. I feel that since both methods are used in test cases, naming this method getLocationsForTest doesnt differentiate the purpose of the 2 methods. Perhaps naming it as getUnorderedTypicalLocations() would be better?

Reply:
Done.